### ISSUE TITLE
" STRESS BELIEFS "

### ISSUE THREAD

**Issue by defaultstarter** [defaultstarter/boombastic#1]
A mental state that causes strong feelings of worry and pressure, this element is commonly seen in BPO Industries. These people has a wide range of emotions to cover the need of other countries. They set expectations and particular goals for their career growth that they were unable of looking it in their health. In addition, the compensation to their lives are miscomforts. It is undetermined by time and abundance of work. And I believe that 100% of them are experiencing stress. 
Do you believe that stress is harmful to your health? If you do, why and why not?
According to a health psychologist, people who experience a lot of stress will increase the risk of dying, for those who believe. How you think about stress matters, becuase when you change your mind about stress you can change your body to respond to it. For example, call center agents in multi-tasking. It sounds simple but they are too much pressured and they are not feeling good about it, then quiet a while they can manage themselves.
Basically, if you believe that stress is bad to experience, make it happen but dont believe that you will be dying. Becuase when you think about it negatively, nothing will oppose to happen. So just be in a full of courage and optimism, something will follow to set things right for you.
